docs/services/slam/cartographer/_index.md

```diff
@@ -214,46 +214,28 @@ Select the **Raw JSON** mode, then copy/paste the following `"services"` and `"m
 
 ### Adjust `data_dir`
 
-Change the `data_dir` attribute to point to a directory on your machine where you want to save the sensor data your SLAM service uses and the maps and config files it produces.
+Change the `data_dir` attribute to point to a directory on your machine where you want to save the internal state your SLAM service produces.
 
 This directory must be structured as follows:
 
 <pre>
 .
 └──\(<file>CARTOGRAPHER_DIR</file>)
-    ├── <file>map</file>
-    ├── <file>data</file>
-    └── <file>config</file>
+    ├── <file>internal_state</file>
 </pre>
 
 Click through the following tabs to see the usage of each folder in this directory:
 
-{{% tabs name="Folders"%}}
-{{% tab name="/map" %}}
-
-Whether mapping data is present in <file>map</file> at runtime and the attribute `map_rate_sec` determines the SLAM mapping mode:
+Whether internal state data is present in <file>internal_state</file> at runtime and the attribute `map_rate_sec` determines the SLAM mapping mode:
```

Comment:
I think this part of the docs is a bit confusing especially as it is rendered in the docs page:
<img width="758" alt="Screenshot 2023-07-27 at 3 41 37 PM" src="https://github.com/viamrobotics/docs/assets/121991867/06918ac4-b3fe-424e-a333-01447019a864">


- Should it be moved under the Mapping Mode header
- Should it be changed to something like `The SLAM mapping mode is determined by 2 conditions: 
        1. If the internal state data is present in <file>internal_state</file> at runtime
        2. the attribute `map_rate_sec`

Reply:
let me know what you think now